src/server/pachyderm_test.go

```diff
@@ -2258,7 +2259,7 @@ func TestManyFilesSingleCommit(t *testing.T) {
 	require.NoError(t, c.CreateRepo(dataRepo))
 
 	// Request enough to require more than one page of results
-	numFiles := 20000
+	numFiles := 5000
```

Comment:
Lower number here? Does this still requre more than one page of results as the above comment suggests?

Reply:
So, I do not think that comment really applies anymore. It looks like the history of that is that there used to be a hardcoded limit on the etcd request size (10,000 kvs), but it is now something that gets, I think, dynamically resized with a potential upper bound of 262,144 https://github.com/pachyderm/pachyderm/blob/f07bb15aed7cd137d0f5920b3c551f24ec66a45f/src/server/pkg/collection/collection.go#L24

Reply:
Yup, that's accurate, I changed that a while ago. Would you mind removing the comment?

Reply:
Sure